src/components/landing/Addmoredetail.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
We may not need a whole component for a single link to be added. So you may jus add the link and handle it in there.
Also, what we need is a link not a button. if you see it on the page it is big and takes too much attention, We want to add additional feature where the user will be able to add more info by clicking on it.